client/components/Cart.jsx

```diff
@@ -1,9 +1,65 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
-const Cart = () => (
-  <div>
-    <h1>CART</h1>
-  </div>
-);
+const propTypes = {
+  cartItems: PropTypes.arrayOf(PropTypes.shape({
+    quantity: PropTypes.number,
+    orderId: PropTypes.number,
+    productId: PropTypes.number,
+  })).isRequired,
+  products: PropTypes.arrayOf(PropTypes.shape({
+    productId: PropTypes.number,
```

Comment:
I think imageUrl and categoryId don't have to be required, might be helpful if the others were required